.kitchen.yml

```diff
@@ -51,3 +51,10 @@ suites:
   - name: default
     run_list:
       - recipe[wrapper::default]
+
+# Can be used as a quick way to test the python bugs on centos 6
+# https://github.com/rackspace-cookbooks/stack_commons/issues/85
+#
+#  - name: python
+#    run_list:
+#      - recipe[stack_commons::python]
```

Comment:
I almost left it in there permanently, since this is the 3rd or 4th time I've had to fix something between chef and python and our CentOS cloud images. I figured it would save some time if it came up a 5th time.